javaslang/src-gen/main/java/javaslang/Tuple3.java

```diff
@@ -124,6 +124,17 @@ public T1 _1() {
     }
 
     /**
+     * Wither of the 1st element of this tuple.
+     *
+     * Note: The wither is NOT named {@code _1} due to possible ambiguity when using method reference (Tuple3::_1).
+     *
+     * @return a copy of this tuple with a new value for the 1st element of this Tuple.
+     */
+    public Tuple3<T1, T2, T3> w1(T1 t1) {
```

Comment:
The [new date-time API has with](http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/8-b132/java/time/ZonedDateTime.java#ZonedDateTime.withMinute%28int%29)s, I think they're less confusing than a `set` (which could imply that the Tuple is mutable)